protobuf-kafkaconnect-converter/src/test/java/com/amazonaws/services/schemaregistry/kafkaconnect/protobuf/ToProtobufTestDataGenerator.java

```diff
@@ -285,6 +288,79 @@ private static Map<String, Schema> getEnumType() {
                 .build();
     }
 
+
+
+    public static Schema getTimeSchema(String name) {
+        return createConnectSchema(name, getTimeTypes(), ImmutableMap.of());
+    }
+
+    @SneakyThrows
+    public static DynamicMessage getProtobufTimeMessage() {
+        Descriptors.FileDescriptor fileDescriptor = getTimeFileDescriptor();
+        Descriptors.Descriptor descriptor = fileDescriptor.getMessageTypes().get(0);
+        DynamicMessage.Builder dynamicMessageBuilder = DynamicMessage.newBuilder(descriptor);
+
+        com.google.type.Date.Builder dateBuilder = com.google.type.Date.newBuilder();
+        dateBuilder.setYear(2022);
+        dateBuilder.setMonth(3);
+        dateBuilder.setDay(20);
+
+        com.google.type.TimeOfDay.Builder todBuilder = com.google.type.TimeOfDay.newBuilder();
+        todBuilder.setHours(2);
+        todBuilder.setMinutes(2);
+        todBuilder.setSeconds(42);
+
+        com.google.protobuf.Timestamp.Builder timestampBuilder = com.google.protobuf.Timestamp.newBuilder();
+        timestampBuilder.setSeconds(1);
+        timestampBuilder.setNanos(805000000);
+
+        return dynamicMessageBuilder
+                .setField(descriptor.findFieldByName("date"), dateBuilder.build())
+                .setField(descriptor.findFieldByName("time"), todBuilder.build())
+                .setField(descriptor.findFieldByName("timestamp"), timestampBuilder.build())
+                .build();
+    }
+
+    private static Descriptors.FileDescriptor getTimeFileDescriptor() {
+        return new ConnectSchemaToProtobufSchemaConverter().convert(getTimeSchema("timeProtobufSchema"));
+    }
+
+    public static Struct getTimeTypeData() {
+        Schema connectSchema = createConnectSchema("timeProtobufSchema", getTimeTypes(), ImmutableMap.of());
+        final Struct connectData = new Struct(connectSchema);
+
+        int dateDefVal = 365 * 52 + 91;
+        int timeDefVal = 1000 * 60 * 60 * 2 + 45 * 60 * 60;
+        long tsDefVal = 30 * 60 + 5;
+        java.util.Date date = Date.toLogical(Date.SCHEMA, dateDefVal);
+        java.util.Date time = Time.toLogical(Time.SCHEMA, timeDefVal);
+        java.util.Date timestamp = Timestamp.toLogical(Timestamp.SCHEMA, tsDefVal);
+
+        connectData
+                .put("date", date)
+                .put("time", time)
+                .put("timestamp", timestamp);
+        return connectData;
+    }
+
+    private static Map<String, Schema> getTimeTypes() {
+//        int dateDefVal = 100;
```

Comment:
Do we need these comments? Also, clean up enum changes in this file.

Reply:
Must have gotten mixed up during the rebase, will remove these.